src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,22 @@
+package duke;
+
+/**
+ * Class for Ui components of Duke.
+ */
+public class Ui {
+
+    public static String divider() {

```

Comment:
I the divider is returned in a method instead of a constant, maybe you can consider renaming the method to make it sound like a verb instead e.g. printDivider.